org_status/org_status.py

```diff
@@ -120,6 +129,17 @@ def main():
 
     hosts_only_print = print if args.hosts_only else verbose
 
+    if args.check_providers_only:
+        for provider, status in get_status_provider_statuses():
+            if provider is not None:
+                # you need to improve this statement to print better
+                # and aptly styled text message
+                print(f'{provider.NAME}: {status}')
+            else:
+                print(f'{provider.NAME} does not support status checking')
```

Comment:
If the `provider` is None, it cannot have the `NAME` attribute. Probably shouldn't yield None.